modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -364,6 +374,45 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private void _hasPreventDeletionType(long objectDefinitionId) {
```

Comment:
I search for the sufix "has" when we are creating a void method and I did not find any example. Could we rename this method? What do you think of "_check" ?

Reply:
So, with this sufix, I think is better call `_checkDeletionType`. Or continue with `_checkPreventDeletionType`?